soc/arm/st_stm32/stm32f0/Kconfig.defconfig.stm32f098xx

```diff
@@ -0,0 +1,18 @@
+# Kconfig - ST Microelectronics STM32F098XX MCU
+#
+# Copyright (c) 2019 The Chromium OS Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32F098XX
+
+config SOC
+	string
```

Comment:
Could remove the type. See note in https://docs.zephyrproject.org/latest/guides/kconfig/index.html#common-shorthands.